addons/mrp/models/mrp_production.py

```diff
@@ -801,7 +820,10 @@ def action_view_mrp_production_sources(self):
     def action_confirm(self):
         self._check_company()
         for production in self:
-            production.consumption = production.bom_id.consumption
+            if not production.bom_id:
```

Comment:
```python
            if production.bom_id:
                production.consumption = production.bom_id.consumption
            else:
                production.consumption = 'flexible'
```

Reply:
wow wow wow
workorders without bom?
i'll ask @oma-odoo what's her opion on that.. but it stinks